### DISCUSSION TITLE
Password generator: enable or disable custom group

### DISCUSSION THREAD

**Discussion by username116** [keepassxreboot/keepassxc#9573] [Ideas]
Hi,

In the password generator, there are **Also choose from** and **Do not include** custom groups. Sometimes there are websites that only support letters and numbers. I would like a way to temporarily disable the **Also choose from** custom group, like the other groups. Or with a checkbox.
Currently, to disable this group, I have to delete all the characters, generate my password, then I have to find my setting again... it would be much easier with a checkbox.

![Capture d'écran 262](https://github.com/keepassxreboot/keepassxc/assets/108369600/5ab99ad6-12a3-499a-87e0-648f37ac62f6)

**Comment by simheap** [comment]
I like this idea. It's simple but truly useful!
Sometimes, I need to disable those custom groups, but I really don't want to copy them to a temporary location and then delete everything in KeePassXC.
